source/_components/light.template.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+title: "Template Light"
+description: "Instructions how to integrate Template lights into Home Assistant."
+date: 2016-05-18 20:32
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_release: 0.46
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `template` platform creates lights that combine components.
+
+For example, if you have a garage door with a toggle switch that operates the motor and a sensor that allows you know whether the door is open or closed, you can combine these into a switch that knows whether the garage door is open or closed. If your garage door also has the ability to where you can set its position, you can bind the `set_level` command to a script or service and capture the brightness variable and manipulate it before sending it to your script or service.
+
+This can simplify the gui, and make it easier to write automations. You can mark the components you have combined as `hidden` so they don't appear themselves.
+
+To enable Template lights in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
```

Comment:
A simpler example for users getting started could be preferable and having the advanced one as an option after the configuration variables.